chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -548,21 +553,26 @@ Relating this back to our classification objective, even if we get the best poss
 Our loss is lower-bounded by the entropy given by the actual conditional distributions $P(\mathbf{y} \mid \mathbf{x})$.
 -->
 
-*dịch đoạn phía trên*
-
+Vì thế nếu entropy là mức độ của sự ngạc nhiên trải nghiệm bởi một người biết xác suất đúng, vậy bạn có thể băn khoăn, *cross-entropy là gì?*
+Cross-entropy *từ $p$ đến $q$*, ký hiệu H(p, q), là sự bất ngờ được kỳ vọng của một người quan sát với các xác suất chủ quan $q$ đối với việc nhìn thấy dữ liệu mà đã được thực sự tạo ra dựa trên các xác suất $p$.
+Khả năng cross-entropy thấp nhất có thể đạt được khi $p = q$.
+Trong trường hợp này, cross-entropy từ $p$ đến $q$ là $H(p, p) = H(p)$.
+Điều này liên quan trở lại mục tiêu phân loại của chúng ta, thậm chí khi chúng ta có khả năng dự đoán tốt nhất có thể, tốt nhất có thể có thể, thì chúng ta sẽ không bao giờ hoàn hảo.
+Mất mát của chúng ta bị giới hạn thấp hơn bởi entropy được cho bởi các phân phối thực tế có điều kiện $P(\mathbf{y} \mid \mathbf{x})$.
 
 <!--
 ### Kullback Leibler Divergence
 -->
 
-### *dịch tiêu đề phía trên*
+### Phân Tán Kullback Leibler
 
 <!--
 Perhaps the most common way to measure the distance between two distributions is to calculate the *Kullback Leibler divergence* $D(p\|q)$.
 This is simply the difference between the cross-entropy and the entropy, i.e., the additional cross-entropy incurred over the irreducible minimum value it could take:
 -->
 
-*dịch đoạn phía trên*
+Có lẽ cách thông thường nhất để đo lường khoảng cách giữa hai phân chia là tính toán *phân tán Kullback Leibler* $D(p\|q)$.
+Điều này đơn giản là sự khác nhau giữa cross-entropy và entropy, điều đó nói rằng, giá trị cross-entropy bổ sung phát sinh so với giá trị tối thiểu không thể giảm được mà nó có thể nhận:
```

Comment:
```suggestion
Phân tán Kullback Leibler đơn giản là sự khác nhau giữa entropy chéo và entropy, có nghĩa là giá trị entropy chéo bổ sung phát sinh so với giá trị tối thiểu không thể giảm được mà nó có thể nhận:
```

Từ `this` hay được dùng phổ biến trong tiếng Anh nhưng tiếng Việt đôi khi không có lợi lắm. Bạn có thể biến đổi sao cho phù hợp hơn